libbeat/outputs/elasticsearch/client.go

```diff
@@ -312,6 +312,10 @@ func createEventBulkMeta(
 	}
 
 	index, err := indexSel.Select(event)
+	if index == "" {
```

Comment:
Let's first check the `err` and then check the index.

Then, please take a look which tests are failing.